ballot/models.py

```diff
@@ -1377,6 +1377,25 @@ def find_closest_ballot_returned(self, text_for_map_search, google_civic_electio
             'ballot_returned':          ballot_returned,
         }
 
+    def should_election_search_data_be_saved(self, google_civic_election_id):
+        success = False
+        status = ""
+        if not google_civic_election_id:
+            status = 'MISSING_GOOGLE_CIVIC_ELECTION_ID-should_election_search_data_be_saved '
+        else:
+            ballot_returned_list_manager = BallotReturnedListManager()
+            ballot_returned_list_count = ballot_returned_list_manager.fetch_ballot_returned_list_count_for_election(
+                google_civic_election_id)
+            if ballot_returned_list_count > 0:
+                success = True
+                status = 'ELECTION SEARCH DATA BE SAVED'
+
+        results = {
+            'success':  success,
+            'status':   status,
+        }
+        return results
+
```

Comment:
Hi Anisha, for a simple function like "should_election_search_data_be_saved" we don't need to  return results. Instead we can just return True or False.